### PR TITLE
Add RPM to set dist macro

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.19
* [ ] 1.18
* [ ] 1.17

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/foreman/dist-macro/dist-macro.spec [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+BuildArch: noarch
+
+%description
+Sets the dist macro to basic distribution and major version.
+
+%install
+install -d -m 755 %{buildroot}/etc/rpm
+cat >> %{buildroot}/etc/rpm/dist-macro.dist << EOF
+%%rhel %{base_release_version}
```
Comment (ekohl) [inline packages/foreman/dist-macro/dist-macro.spec]: Where do these values come from? Is it to copy from RHEL's release?

Comment (ehelms) [reply packages/foreman/dist-macro/dist-macro.spec]: Look at the top of the spec file.

Comment (ekohl) [reply packages/foreman/dist-macro/dist-macro.spec]: I meant to comment on the line below. I don't see `%dist` and I think it's not great to depend on the build env for it.

**Comment by ehelms** [comment]
To answer your question about centos-release, at the top of the spec they define this:

```
%define dist .el%{dist_release_version}.centos
```

**Comment by ekohl** [comment]
This was changed recently, see https://bugs.centos.org/view.php?id=14955. Using a recent centos-release version will remove the need for this package. At least, that's the theory :)

**Comment by ehelms** [comment]
Updating Koji with centos updates repository and will re-test builds.

**Pull request closed without merging**